multy_core/src/EOS/EOS_transaction.cpp

```diff
@@ -0,0 +1,614 @@
+        {
+            data << *m_memo;
+        }
+
+        return make_clone(data.get_content());
+    }
+
+
+public:
+    EOSTypeName m_from;
+    EOSTypeName m_to;
+    BigInt m_amount;
+    BinaryDataPtr m_memo;
+    std::vector<EOSAuthorization> m_authorizations;
+};
+
+class EOSTransactionSource : public TransactionSourceBase
+{
+public:
+    explicit EOSTransactionSource(BlockchainType blockchain_type)
+        : address(m_properties, "address", Property::REQUIRED,
+                [blockchain_type](const std::string& new_address) {
+                    get_blockchain(BLOCKCHAIN_EOS)
+                            .validate_address(blockchain_type,
+                                    new_address.c_str());
+                }),
+          amount(m_properties, "amount", Property::OPTIONAL,
+                [](const BigInt& new_amount)
+                {
+                    if (new_amount < BigInt(0))
+                    {
+                        THROW_EXCEPTION("Can't set negative amount as"
+                                " transaction source value.");
+                    }
+                })
+    {}
+
+public:
+    PropertyT<std::string> address;
+    PropertyT<BigInt> amount;
+};
+
+class EOSTransactionDestination : public TransactionDestinationBase
+{
+public:
+    explicit EOSTransactionDestination(BlockchainType blockchain_type)
+        : address(m_properties, "address", Property::REQUIRED,
+                [blockchain_type](const std::string& new_address) {
+                    get_blockchain(BLOCKCHAIN_EOS)
+                            .validate_address(blockchain_type,
+                                    new_address.c_str());
+                }),
+          amount(m_properties, "amount", Property::OPTIONAL,
+                [](const BigInt& new_amount)
+                {
+                    if (new_amount < BigInt(0))
+                    {
+                        THROW_EXCEPTION("Can't set negative amount as"
+                                " transaction destination value.");
+                    }
+                })
+    {}
+
+public:
+    PropertyT<std::string> address;
+    PropertyT<BigInt> amount;
+};
+
+EOSTransaction::EOSTransaction(const Account& account)
+    : TransactionBase(account.get_blockchain_type()),
+      m_account(account),
+      m_message(new BinaryData{nullptr, 0}),
+      m_source(),
+      m_destination(),
+      m_explicit_expiration(
+            get_transaction_properties(),
+            "expiration",
+            Property::OPTIONAL,
+            [this](const std::string& new_expiration)
+            {
+                this->set_expiration(new_expiration);
+            }),
+      m_ref_block_num(
+            get_transaction_properties(),
+            "block_num",
+            Property::REQUIRED),
+      m_ref_block_prefix(
+            get_transaction_properties(),
+            "ref_block_prefix",
+            Property::REQUIRED,
+            [](const BigInt& block_prefix)
```

Comment:
Please add function similar to `verify_bigger_than` but name it `verify_smaller_than`